runtime/CSharp/runtime/CSharp/Antlr4.Runtime/Atn/PredictionMode.cs

```diff
@@ -44,7 +44,9 @@ namespace Antlr4.Runtime.Atn
     /// utility methods for analyzing configuration sets for conflicts and/or
     /// ambiguities.
     /// </remarks>
+#if !DOTNETCORE
```

Comment:
:bulb: You can update [SerializableAttribute.cs](https://github.com/antlr/antlr4/blob/master/runtime/CSharp/runtime/CSharp/Antlr4.Runtime/Sharpen/Compat/SerializableAttribute.cs) to avoid all these directives.